scripts/current_validators

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/python3
+
```

Comment:
If the scripts need to be installed with the node package, they should be added to `setup.py` (`scripts` field).